src/encord_active/lib/common/project.py

```diff
@@ -27,52 +27,64 @@
 
 
 class Project:
-    def __init__(self, project_dir: Path, subset_size: Optional[int] = None):
+    def __init__(self):  # Project skeleton, intended only for internal usage
+        self.project_dir: Path
+        self.project_meta: Dict[str, str]
+        self.project_hash: str
+        self.ontology: OntologyStructure
+        self.label_row_meta: Dict[str, LabelRowMetadata]
+        self.label_rows: Dict[str, LabelRow]
+        self.image_paths: Dict[str, List[Path]]
+
+    @classmethod
+    def load(cls, project_dir: Path, subset_size: Optional[int] = None):
```

Comment:
Maybe like this instead, wdyt?

```suggestion
    def __init__(self, project_dir: Optional[Path] = None):
        self.project_dir = project_dir
        self.project_meta: Dict[str, str]
        self.project_hash: str
        self.ontology: OntologyStructure
        self.label_row_meta: Dict[str, LabelRowMetadata]
        self.label_rows: Dict[str, LabelRow]
        self.image_paths: Dict[str, List[Path]]

    def load(self, subset_size: Optional[int] = None):
```

This way you don't need `fake_project` and you can do `Project("/some/path").load()`